redis/src/cluster_client.rs

```diff
@@ -393,6 +399,16 @@ impl ClusterClientBuilder {
         self
     }
 
+    /// Enables periodic connections checks for this client.
+    /// If enabled, the conenctions to the cluster nodes will be validated periodicatly, per configured interval.
+    /// In addition, for tokio runtime, passive disconnections could be detected instantly,
+    /// triggering reestablishemnt, w/o waiting for the next periodic check.
+    #[cfg(feature = "cluster-async")]
+    pub fn periodic_connections_checks(mut self, interval: Duration) -> ClusterClientBuilder {
```

Comment:
Nit: the `Rust` convention is to use the `with_` prefix for build patterns
Nit2: the function should match the param (or vice versa): e.g. `with_connection_validation_interval`

Reply:
Yep, but aligning with the existing style is more important (note periodic_topology_checks)